backend/infrahub/core/migrations/graph/m010_add_generate_profile_attr_generic.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from typing_extensions import Self
+
+from infrahub.core.constants import SchemaPathType
+from infrahub.core.migrations.shared import MigrationResult
+from infrahub.core.path import SchemaPath
+
+from ..schema.node_attribute_add import NodeAttributeAddMigration
+from ..shared import InternalSchemaMigration
+
+if TYPE_CHECKING:
+    from infrahub.database import InfrahubDatabase
+
+
+class Migration010(InternalSchemaMigration):
+    name: str = "010_add_generate_profile_attr_generic"
+    minimum_version: int = 9
+
+    @classmethod
+    def init(cls, *args: Any, **kwargs: dict[str, Any]) -> Self:
```

Comment:
The `*args` were removed from the base .init() method and I think it was never used. See that it came back here and in migration 090.